.github/workflows/integration_test.yaml

```diff
@@ -8,8 +8,10 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
-      extra-arguments: --s3-url http://172.17.0.1:4566
-      pre-run-script: localstack-installation.sh
+      chaos-app-label: app.kubernetes.io/name=discourse-k8s
       chaos-enabled: true
       chaos-experiments: pod-delete
-      chaos-app-label: app.kubernetes.io/name=discourse-k8s
+      chaos-duration: 180
+      extra-arguments: --s3-url http://172.17.0.1:4566
+      pre-run-script: localstack-installation.sh
+      trivy-image-config: "trivy.yaml"
```

Comment:
Supplied a `trivy.yaml` config file like we do on Indico and there's an open issue about addressing the found vulnerabilities.